src/components/Confirm/index.js

```diff
@@ -57,35 +58,48 @@ class CaptureViewer extends Component {
   }
 }
 
-
-const Previews = ({capture, retakeAction, confirmAction} ) =>
-  <div className={`${theme.previews} ${theme.step}`}>
+const PreviewHeader = () =>
+  <div>
     <h1 className={theme.title}>Confirm capture</h1>
     <p>Please confirm that you are happy with this photo.</p>
-    <CaptureViewer capture={capture} />
-    <div className={`${theme.actions} ${style.actions}`}>
-      <button
-        onClick={retakeAction}
-        className={`${theme.btn} ${style["btn-outline"]}`}
-      >
-        Take again
-      </button>
-      <a
-        href=''
-        className={`${theme.btn} ${theme["btn-primary"]}`}
-        onClick={preventDefaultOnClick(confirmAction)}
-      >
-        Confirm
-      </a>
+  </div>
+
+const RetakeAction = ({retakeAction}) =>
+  <button onClick={retakeAction}
+    className={`${theme.btn} ${style["btn-outline"]}`}>
+    Take again
+  </button>
+
+const ConfirmAction = ({confirmAction, error}) => {
+  if (!error) return (
+    <a href='' className={`${theme.btn} ${theme["btn-primary"]}`}
```

Comment:
Oh, yes I will fix that. The code for the component was already there and I didn't noticed it had an empty string instead of a `#` for the href. I'll fix it.